Kentico.Kontent.Delivery.Extensions.Autofac.DependencyInjection/DeliveryClientFactory.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Autofac;
+using Autofac.Core.Registration;
+using Kentico.Kontent.Delivery.Abstractions;
+using Kentico.Kontent.Delivery.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Concurrent;
+
+namespace Kentico.Kontent.Delivery.Extensions.Autofac.DependencyInjection
+{
+    internal class DeliveryClientFactory : IDeliveryClientFactory
```

Comment:
for the sake of simplicity, I'd rename this implementation of IDeliveryClientFactory to something that cannot be mistaken with the default implementation...so what about `NamedDeliveryClientFactory`

Reply:
Renamed